src/StoreKeeper/WooCommerce/B2C/Tools/TaskRateCalculator.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace StoreKeeper\WooCommerce\B2C\Tools;
+
+use StoreKeeper\WooCommerce\B2C\Models\TaskModel;
+
+class TaskRateCalculator
+{
+    private $tasks;
+    private $minuteMetrics;
+    private $startDateTime;
+    private $endDateTime;
```

Comment:
Why don't you use the [unit timestamp](https://www.unixtimestamp.com/) here? It would make the calculation way easier. You can still accept the dates in the constructor just convert it there.

Reply:
We are not using these for calculation but for comparison for dates in sql.